tests/postgres_tests/test_aggregates.py

```diff
@@ -146,6 +146,16 @@ def test_string_agg_distinct_true(self):
         self.assertEqual(values['stringagg'].count('Bar'), 1)
 
 
+class TestArrayAggregateDistinct(TestStringAggregateDistinct):
```

Comment:
Using a subclass isn't correct. That will cause the tests in the super class to be executed again.